src/admin-views/settings/emails/message-templates/list-item.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * Tickets Emails Message Template List Item
+ *
+ * @since  TBD  List item of email message templates for Emails settings tab.
+ * 
+ * @var Array[]  $templates  Array of template info.
+ * @var Array    $template   Template info.
+ */
+
+// @todo $templates variable will be an array of Message_Template objects in the future.
+// @todo $template variable will be a Message_Template object in the future.
+
+// If no templates, bail.
+if ( empty( $template ) ) {
+	return;
+}
+
+$item_classes   = [ 'tec_tickets-emails-template-list-item' ];
+$item_classes[] = tribe_is_truthy( $template['enabled'] ) ? 'enabled' : 'disabled';
```

Comment:
To follow BEM we should use a modifier for this:

```
.tec_tickets-emails-template-list-item--enabled
.tec_tickets-emails-template-list-item--disabled
```

Please check the comment about class naming in the pcss file.